pkg/project/auth/cache_test.go

```diff
@@ -0,0 +1,143 @@
+package auth
+
+import (
```

Comment:
Ok, I found pods, but it's still disconnected.  Is simply listing good enough or do we need it wired together like the registry mocks: https://github.com/deads2k/origin/blob/deads-prevent-escalation/pkg/authorization/registry/test/policy.go ?